app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -101,6 +102,14 @@
             app:singleLineTitle="false"
             app:iconSpaceReserved="false" />
 
+        <SwitchPreferenceCompat
```

Comment:
> Again, please don't introduce new settings. I'd just have this enabled in debug builds (or you can move the setting to debug) and disabled in normal builds for now. This is not a video-audio setting, this is a developer setting, the normal user should not have anything to do with it.

Why is introducing new settings a problem?
Yes this is e.g. for debugging purposes, but I don't understand why we should hide it from the user.
We may agree upon that I move the setting to the debug section. However then the debug section should also be available in the release app.

> I'd just have this disabled in normal builds since having an error popup may be disturbing.

It is disabled by default.

Btw the settings need a major overhaul: #3192

Reply:
> Why is introducing new settings a problem?

If each PR added a new setting to toggle the behavior it is introducing we would have thousands of settings, of which 95% are mostly unused and the other useful 5% is difficult to find and is unknown by most users, who would not go through all thousand settings just to find out what's of interest to them. So if customizing the app becomes too difficult (and also risky I'd say, because there would be many more untested combinations), less and less people are going to really dive into it, defeating the purpose of customizability. Ad I said, the normal user does not want to see a debug setting in the video-audio settings (which is already cluttered), since it makes it difficult to find all of the others. That's why I am opposed to adding new settings unless they would be really useful for the normal user.

Therefore, for now I'd not add a new setting in normal builds and just leave the option disabled. After this PR is merged I will do an overhaul of the error handling for services, so that errors are first posted as notifications and if the user taps on them he gets to the error activity. But if the user just wants to ignore error notifications (like he can ignore the error snackbar) he can just swipe it away or even disable that notification channel in the system settings directly.